api/auth/token.go

```diff
@@ -14,15 +14,15 @@ import (
 
 var ErrInvalidToken = errors.New("invalid token")
 
-func GenerateToken(userID int32, secret, jwt_audience string) (string, error) {
+func GenerateToken(userID int32, role string, secret, jwt_audience string) (string, error) {
 	expires := time.Now().Add(time.Hour * 8).Unix()
 	notBefore := time.Now().Unix()
 	issuer := "https://www.bestqa.net"
 
 	claims := jwt.MapClaims{
 		"user_id": strconv.FormatInt(int64(userID), 10),
 		"exp":     expires,
-		"role":    "user",
+		"role":    role,
 		"jti":     uuid.Must(uuid.NewV4()).String(),
 		"iss":     issuer,
 		"nbf":     notBefore,
```

Comment:
:robot: OpenAI

This patch is acceptable. It adds an additional parameter `role` to the `GenerateToken` function, which makes it more flexible and allows for different roles to be assigned to users.

<!-- This is an auto-generated comment by OpenAI -->